### ISSUE TITLE
NTR: Bone marrow: Gaucher cells (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9131]
New term request Bone marrow: Gaucher cells (segal_100513205937): Lipid-laden macrophages with crumpled silk appearance  UMLS: 1856530

**Comment by pnrobinson** [comment]
@MickeySegal

Gaucher cells are not limited to the bone marrow, e.g. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4451492/
I would propose to make one term underneath [Abnormal cell morphology HP:0025461](https://hpo.jax.org/app/browse/term/HP:0025461) rather than one per organ, because I do not think the localization matters clinically.
Thoughts?

**Comment by MickeySegal** [comment]
That is fine.  We list Gaucher cells only under bone marrow because that is apparently the recommended biopsy.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000893
- label: Gaucher cells
- parents: [Abnormal cell morphology (HP:0025461)](https://hpo.jax.org/app/browse/term/HP:0025461)
- pmid: PMID:28218669
- def: Glucosylceramide forms fibrillar aggregates that accumulate in macrophages and result in the cell cytoplasm presenting a characteristic crumpled tissue paper appearance. These cells, known as Gaucher cells, infiltrate various organs (e.g., bone marrow, spleen, and liver) and are responsible for the major signs of Gaucher disease.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9131
- synonyms:
